prow/config.yaml

```diff
@@ -77,6 +77,48 @@ presubmits:
       - name: cache-ssd
         hostPath:
           path: /mnt/disks/ssd0
+    run_after_success:
+    - name: pull-kubernetes-e2e-kubeadm-gce
+      context: Jenkins kubeadm e2e
+      always_run: false
+      skip_report: true
+      rerun_command: "@k8s-bot kubeadm e2e test this"
+      trigger: "@k8s-bot (kubeadm e2e )?test this"
```

Comment:
Cool, sounds like a plan to me :)
Just wasn't sure if you had noticed the possibility to do that